full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -56,26 +56,31 @@ export namespace AzuriteTest {
       return blobClient.exists();
     };
 
-    export const overwriteWriteLockValue = async (container: CloudSqlite.CloudContainer, blockName: string) => {
+    export const setWriteLockToExpired = async (container: CloudSqlite.CloudContainer, blockName: string) => {
       const azClient = createAzClient(container.containerId);
       const blobClient = azClient.getBlockBlobClient(blockName);
       const tempFilePath = join(KnownLocations.tmpdir, blockName);
+      // download bcv_kv.bcv to local machine
       await blobClient.downloadToFile(tempFilePath);
-
-      const bcv_kv = await open({
-        filename: tempFilePath,
-        driver: sqlite3.Database,
+      const bcv_kv = new SQLiteDb();
+      bcv_kv.openDb(tempFilePath, {openMode: OpenMode.ReadWrite, rawSQLite: true});
+      // read writeLock.expires and modify it
+      bcv_kv.withSqliteStatement("SELECT v FROM kv WHERE k = 'writeLock'", (stmt) => {
+        if (stmt.nextRow()) {
+          const writeLockData = JSON.parse(stmt.getValueString(0));
+          const expiresDate = new Date(writeLockData.expires);
```

Comment:
Rereading this more closely, this is actually subtracting 5 minutes from the expiry date of the write lock right? I thought at first it was doing current time - 5 mins to guarantee expired. So my naming suggestion might be wrong in that case. Maybe it could be changed to do current time - 5 minutes? Or maybe a parameter that says numMinsToSubtract?

Reply:
m_lockExpireSeconds = std::min((int) (12*SECONDS_PER_HOUR), std::max(5*(int)SECONDS_PER_MINUTE, m_lockExpireSeconds));

this is the formula we use to calculate m_lockExpireSeconds . In our test case, it is 5mins.
we also have GetServerTime() on native to read/set the expires time in bcv_kv.bcv. 
According to the comment of GetServerTime:

   * Get the date(s) returned by the server in the http header when the bcv_kv blob was fetched.
   * This avoids any dependence on the local computer's time for detecting expired write locks.

If we do current time -5 mins, then there's a possibility that (current time -5 mins) > GetServerTime(), it will prevent user2 from grabbing the write lock.

Reply:
Since we use azurite, a local storage emulator I would expect that the server's time is also the local computer's time. I only want the function name to accurately represent what its doing. The name 'makewritelockexpired' is only true in the specific case that the write lock is already grabbed for 5 minutes.

Reply:
Resolved in [commit 1ea569504b20baaa08796b742a7521f363f75423](https://github.com/iTwin/itwinjs-core/pull/7000/commits/1ea569504b20baaa08796b742a7521f363f75423)